scripts/st2bootstrap-deb.sh

```diff
@@ -96,6 +96,33 @@ setup_args() {
   fi
 }
 
+check_st2_host_dependencies() {
+  # Check that the following TCP ports are available.
+  # Abort the installation early if the required ports are being used by an existing process.
+
+  # nginx (80, 443), mongodb (27017), rabbitmq (4369, 5672, 25672),
+  # postgresql (5432) and st2 (9100-9102).
+
+  # NOTE: lsof restricts the number of ports specified with "-i" to 100.
+  echo "Checking if required TCP ports are already in use."
+  ret=`sudo /usr/bin/lsof -V -P -i :80 -i :443 -i :4369 -i :5432 -i :5672 -i :9100 -i :9101 -i :9102 \
```

Comment:
Just a thought - any considerations around IPv6 here? `lsof` reports sockets, so if a program was listening on one of these ports via IPv6, it would still show up in your query, but if stackstorm doesn't care about v6 sockets, it may not be a problem. Just something to think about. Maybe just add "IPv4" to your grep (easy way out) or make a separate call to distinguish between the two.